store/src/sql/access/event.rs

```diff
@@ -295,48 +295,57 @@ impl CeramicOneEvent {
         Ok((max, rows))
     }
 
-    /// Returns the root CIDs and the data of all the events found after the given delivered value.
+    /// Returns the root CIDs and values of all the events found after the given delivered value.
     pub async fn new_events_since_value_with_data(
         pool: &SqlitePool,
-        delivered: i64,
+        highwater: i64,
         limit: i64,
-    ) -> Result<(i64, Vec<(Cid, unvalidated::Event<Ipld>)>)> {
+    ) -> Result<(i64, Vec<(Cid, unvalidated::Event<Ipld>, i64)>)> {
```

Comment:
Can you change this to use a struct instead of a tuple? Its confusing what the different i64s are for. I get it after reading the code, however a struct would make it more clear.